test/entry.ts

```diff
@@ -16,22 +16,27 @@
 
 'use strict';
 
-const assert = require('assert');
-const extend = require('extend');
-const {GrpcService, util} = require('@google-cloud/common-grpc');
-const proxyquire = require('proxyquire');
+import * as assert from 'assert';
+import * as extend from 'extend';
+import {Service, util} from '@google-cloud/common-grpc';
```

Comment:
I'll assume that the semantic change from `GrpcService` to `Service` is okay. It is usually better to avoid such semantic deltas in refactoring PRs.